tokio/build.rs

```diff
@@ -117,6 +161,14 @@ fn main() {
         autocfg::emit("tokio_no_const_mutex_new")
     }
 
+    if !target_needs_atomic_u64_fallback {
+        // To disable this feature on compilers that support it, you can
+        // explicitly pass this flag with the following environment variable:
+        //
+        // RUSTFLAGS="--cfg tokio_no_atomic_u64"
+        autocfg::emit("tokio_no_atomic_u64")
+    }
```

Comment:
It seems like the boolean is inverted here. Surely we would only disable atomic u64 if we need the fallback?

Reply:
For some reason, when I do this, it looks like the FreeBSD checks fail?

Reply:
(CI issue is maybe related to https://www.githubstatus.com/incidents/4p7zympgsr86)